src/patternfly/components/Select/select-menu-input.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="pf-c-select__menu-input{{#if select-menu-input--modifier}} {{select-menu-input--modifier}}{{/if}}">
+  {{#> form-control controlType="input" form-control--modifier="pf-m-search" form-control--attribute=(concat 'type="search" id="' id '-search-input" name="' id '-search-input" aria-label="Search"')}}
```

Comment:
What if in the future we want to put a different type of input into this wrapper? I guess we could leave this and then add a condition inside of here to either show search or a different type of input, wdyt?

Reply:
Love this comment/suggestion. **Nice**!!!